src/components/DiscriminationFreeSpaces.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <section>
```

Comment:
You need a "master" class for the component as a whole. In this case, keeping the naming convention you're using it would be `discrimination-free-spaces`. Personally, I'd shorten it because it's too long.